include/ofi_shm.h

```diff
@@ -150,6 +150,8 @@ struct smr_cmd {
 #define SMR_COMP_INJECT_SIZE	(SMR_INJECT_SIZE / 2)
 
 #define SMR_NAME_SIZE	NAME_MAX
+char smr_ep_name[SMR_NAME_SIZE];
```

Comment:
This is unrelated to this PR, but maybe we should just drop SMR_NAME_SIZE and use NAME_MAX in its place.

Reply:
Use of a global here assumes that the app only allocates a single endpoint.  We shouldn't make that assumption.  Does shm have this restriction?

Reply:
That's fair, I can create a global list of endpoints for this process and use that for the cleanup from within the handler.